awx/lib/awx_display_callback/events.py

```diff
@@ -165,11 +162,11 @@ def get_end_dict(self):
         return {}
 
     def dump(self, fileobj, data, max_width=78, flush=False):
-        b64data = base64.b64encode(json.dumps(data))
+        b64data = base64.b64encode(json.dumps(data).encode('utf-8')).decode()
```

Comment:
I don't know much about the code, but just checking the return of `dumps`, it seems the encode is not necessary since it will probably return an UTF-8 string:

```python
In [26]: r = json.dumps({"name": "elyézer"})                                                                       

In [27]: r                                                                                                         
Out[27]: '{"name": "ely\\u00e9zer"}'

In [28]: r.encode()                                                                                                
Out[28]: b'{"name": "ely\\u00e9zer"}'
```

Reply:
I see, in that case it may be necessary for some cases. Thank you for providing more information.